src/GifPickerReact.css

```diff
@@ -5,8 +5,8 @@
     --gpr-text-color: #858585;
     --gpr-search-input-bg-color: #f6f6f6;
     --gpr-picker-border-color: #e7e7e7;
+    --gpr-body-category-font-color: #e7e7e7;
```

Comment:
--gpr-category-font-color = --gpr-body-category-font-color = --gpr-picker-border-color there is no point in another redundant varable

Reply:
removed